plugins/cache.rst

```diff
@@ -103,7 +104,24 @@ removed from the cache::
         'cache_lifetime' => 86400*365, // one year
     ];
 
+Caching of different request methods
+````````````````````````````````````
 
+Most of the time you should not change the ``methods`` option. However if you are working for example with HTTPlug
+based SOAP client you might want to additionally enable caching of ``POST`` requests::
+
+    $options = [
+        'methods' => ['GET', 'HEAD', 'POST'],
+    ];
+
+The ``methods`` setting overrides the defaults. If you want to keep caching ``GET`` and ``HEAD`` requests, you need
+to include them. You can specify any request method that conforms to RFC-7230. Request methods are case sensitive,
```

Comment:
https://github.com/php-http/cache-plugin/pull/24 was now changed to force upper case. please change this line to say that.